docs/Home.md

```diff
@@ -29,25 +29,33 @@ and needs to be adjusted when using a lower version.
+
 As mentioned in the features section, devenv is shipped with Xdebug. You need to activate it to actually debug something. For the Frontend, you can use the [Xdebug helper](https://chrome.google.com/webstore/detail/xdebug-helper/eadndfjplgieldjbigjakmdgkmoaaaoc?gl=US&hl=en) extension when using Chrome. Alternatively, you can simply add a query parameter to the URL you want to debug. Like this: `https://127.0.0.1:8000/?XDEBUG_SESSION=1`
 This also works great when having to debug API requests, e.g. in Insomnia.
 
 For the CLI, you can use the integrated debug helper `debug` by simply adding it before your CLI commands, e.g. `debug bin/console`.
 
-
 # MySQL Configuration
 
 The configuration is based on recommendations from Shopware. You can find the documentation [here](https://developer.shopware.com/docs/guides/hosting/performance/performance-tweaks.html#mysql-configuration).
 By default, we change the following configurations:
 
-| setting              | value                                                                                              | description                                                                                                                 |
+| Setting              | Value                                                                                              | description                                                                                                                 |
 |----------------------|----------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------|
 | group_concat_max_len | 32000                                                                                              | Shopware highly uses GROUP_CONCAT functions within the Database. This setting ensures, that all queries will run correctly. |
 | key_buffer_size      | 16777216                                                                                           | Defines, how many index buffers can be held in memory                                                                       |
 | max_allowed_packet   | 134217728                                                                                          | Defines, how much data can be send within one request before the server blocks it.                                          |
 | table_open_cache     | 1024                                                                                               | Shopware uses many tables, This settings allows the server to keep more tables open                                         |
 | sql_mode             | STRICT_TRANS_TABLES,NO_ZERO_IN_DATE,NO_ZERO_DATE,ERROR_FOR_DIVISION_BY_ZERO,NO_ENGINE_SUBSTITUTION | the default from MySQL, but without ONLY_FULL_GROUP_BY                                                                      |
 
+# Built in Commands
```

Comment:
```suggestion
# Built in commands
```

Reply:
I'd also like to see an example of the execution of a command, e.g. `devenv shell caddy-trust`

Reply:
done